### DISCUSSION TITLE
Turtle Doesn't Work in Editor Tab

### DISCUSSION THREAD

**Discussion by ttrickel** [brython-dev/brython#2464] [Q&A]
TL;DR Is it possible to run Turtle in the black window of the Editor tab?

It's great to see that Brython includes a Turtle Graphics package.

Unfortunately the example code in the Gallery tab doesn't work in the Editor tab. Is this sort of by design?

When I copy and paste the Turtle demo square with default turtle gallery code into the Editor and run the script I get the following error:
Traceback (most recent call last):
  File <string>, line 3, in <module>
KeyError: turtle-div

I suspect this is because the HTML code for the Editor tab doesn't include <div id="turtle-div" section.

If I change the 'turtle-div' in the turtle_canvas_wrapper line to 'main_container' turtle does run but the turtle draws above the editor. 

Could the page source of the Editor tab be modified to include a div for the black window?

Thanks in advance
